server/api/routes-env.js

```diff
@@ -0,0 +1,19 @@
+/* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -  */
+/*  Routes to handle Deployment Environment Variables exposition                                  */
+/* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -  */
+const koaRouter = require('koa-router')
+const consts = require('../utils/consts')
+
+const router = koaRouter({
+  prefix: consts.BASE_API
+})
+
+router.get('/env', async ctx => {
```

Comment:
That way, we could have a multi-level deployment and have Nuxt read from Side-Car, instead.

Note that I couldn't make @nuxtjs/dotenv work properly from Nuxt side.

Sure thing, when managing many deployments of the same app, and dynamic monitoring system, deployment specific toggles could be discovered from here.

What's your opinion, @clarkdo?